manifest.json

```diff
@@ -69,23 +69,63 @@
         "helpMarkDown": "[Get Json Key File](https://console.cloud.google.com/iam-admin/serviceaccounts)",
         "url": {
           "displayName": "Server Url",
-          "value": "https://accounts.google.com/o/oauth2"
+          "value": "https://www.googleapis.com/compute"
         },
-        "inputDescriptors": [],
+        "inputDescriptors": [
+          {
+            "id": "projectid",
+            "name": "i18n:projectid",
+            "description": "i18n:ProjectId for connecting to the endpoint",
+            "inputMode": "textbox",
+            "isConfidential": false,
+            "validation": {
+              "isRequired": true,
+              "dataType": "string",
+              "maxLength": 300
+            }
+          }
+        ],
+        "dataSources": [
+          {
+            "name": "ZoneName",
+            "endpointUrl": "{{{endpoint.url}}}/beta/projects/{{{endpoint.projectid}}}/zones",
+            "resultSelector": "jsonpath:$.items[*].name"
+          },
+          {
+            "name": "MachineName",
+            "endpointUrl": "{{{endpoint.url}}}/beta/projects/{{{endpoint.projectid}}}/zones/$(zone)/instances",
+            "resultSelector": "jsonpath:$.items[*].name"
+          },
+          {
+            "name": "RegionName",
+            "endpointUrl": "{{{endpoint.url}}}/v1/projects/{{{endpoint.projectid}}}/regions",
+            "resultSelector": "jsonpath:$.items[*].name"
+          },
+          {
+            "name": "ProjectName",
+            "endpointUrl": "https://cloudresourcemanager.googleapis.com/v1beta1/projects",
+            "resultSelector": "jsonpath:$.projects[*].projectId"
+          },
+          {
+            "name": "ClusterName",
+            "endpointUrl": "https://container.googleapis.com/v1/projects/{{{endpoint.projectid}}}/zones/-/clusters",
```

Comment:
It would probably be better to filter the clusters list by the zone.

`.../zones/$(zone)/clusters`.

Is there a way to have a default if $(zone) is not set? `.../zones/$(($zone, "-" -ne $null)[0])/clusters`?